pkg/genericapiserver/validation/universal_validation.go

```diff
@@ -17,48 +17,67 @@ limitations under the License.
 package validation
 
 import (
+	"fmt"
+
 	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/genericapiserver/options"
+	utilerrors "k8s.io/kubernetes/pkg/util/errors"
 )
 
 // TODO: Longer term we should read this from some config store, rather than a flag.
-func verifyClusterIPFlags(options *options.ServerRunOptions) {
+func verifyClusterIPFlags(options *options.ServerRunOptions) error {
+	errors := []error{}
 	if options.ServiceClusterIPRange.IP == nil {
-		glog.Fatal("No --service-cluster-ip-range specified")
+		errors = append(errors, fmt.Errorf("No --service-cluster-ip-range specified"))
 	}
 	var ones, bits = options.ServiceClusterIPRange.Mask.Size()
 	if bits-ones > 20 {
-		glog.Fatal("Specified --service-cluster-ip-range is too large")
+		errors = append(errors, fmt.Errorf("Specified --service-cluster-ip-range is too large"))
 	}
+	return utilerrors.NewAggregate(errors)
```

Comment:
This will create nested aggregates. I suggest returning []error from these functions.